Library/ViewModels/LiveStreamDiscoveryViewModel.swift

```diff
@@ -0,0 +1,99 @@
+import KsApi
+import LiveStream
+import Prelude
+import ReactiveSwift
+import Result
+
+public protocol LiveStreamDiscoveryViewModelInputs {
+  /// Call from parent controller when this view is shown to the user.
+  func isActive(_ active: Bool)
```

Comment:
it's a lil different from `configureWith` cause it will be called multiple times, e.g. if you switch away and come back.